src/transform.ts

```diff
@@ -101,6 +102,60 @@ export interface AggregatedFieldDef {
   as: string;
 }
 
+export interface WindowFieldDef {
+  /**
+   * The operations supported for the window aggregation. See the list of supported operations here:
+   *   https://vega.github.io/vega-lite/docs/transforms/window.html
+   */
+  op: AggregateOp | WindowOnlyOp;
+
+  /**
+   *  Parameter values for the window functions. Parameter value can be null for operations that do not accept a
```

Comment:
"See the list of operations for each operation's supported parameter."?

Reply:
Also, for operations that do not accept a parameter -- should param really be `null`?   Why not `undefined` (omitted)?

Reply:
That was my mistake. I copied the definition from the vega docs, but here it can just be omitted. Updated to remove that.